package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "paper-vertical-tabs",
+  "version": "1.0.0",
+  "description": "paper vertical tabs",
+  "main": "paper-vertical-tabs.js",
+  "scripts": {
+    "start": "./node_modules/polyserve/bin/polyserve"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/AleksandrZhukov/paper-vertical-tabs.git"
```

Comment:
Hi @AleksandrZhukov  can you please change this to:

`"url": "git+https://github.com/Battlefy/paper-vertical-tabs.git"`